integration/specs/Tree/tree-1.spec.js

```diff
@@ -15,49 +15,65 @@ describe('Tree basic', () => {
 
     it('should expand the node when it is collapse and its button icon is clicked', () => {
         const tree = new PageTree(TREE);
-        const node = tree.getItem(3);
-        node.clickButtonIcon();
+        const node = tree.getNode(3);
+        node.click();
         expect(node.isExpanded()).toBe(true);
     });
     it('should collapse the node when it is expand and its button icon is clicked', () => {
         const tree = new PageTree(TREE);
-        const node = tree.getItem(2);
-        node.clickButtonIcon();
+        const node = tree.getNode(2);
+        node.click();
         expect(node.isExpanded()).toBe(false);
     });
     it('should move focus to the next button icon when the first button icon is focused and press tab', () => {
         const tree = new PageTree(TREE);
-        const firstNode = tree.getItem(2);
-        const secondNode = tree.getItem(3);
-        firstNode.clickButtonIcon();
+        const firstNode = tree.getNode(2);
+        const secondNode = tree.getNode(3);
+        firstNode.click();
         browser.keys(TAB_KEY);
-        expect(secondNode.hasFocusButtonIcon()).toBe(true);
+        expect(secondNode.hasFocus()).toBe(true);
     });
+
+    /**
+     * The node was initially expanded
+     */
```

Comment:
sorry, I mean in the test name like:
`should expand the node when its button icon is focused, press enter and the node was initially expanded`